src/crystal/system/unix/getrandom.cr

```diff
@@ -38,6 +38,13 @@ module Crystal::System::Random
     end
   end
 
+  private def self.has_sys_getrandom
```

Comment:
This PR is fine as is, but I wonder if it might be clearer to tie the method name together with the class variable name to show they are related.  Something like `private def self.getrandom_available`

Or perhaps even better, just encapsulate the setting of `@@getrandom_available` directly in that method.  Something like:

```crystal
private def self.getrandom_available
  sys_getrandom(Bytes.new(16))
  @@getrandom_available = true
rescue
  @@getrandom_available = false
end
```

Reply:
Or this (shorter & memoized):

```cr
class_getter(getrandom_available : Bool) { !!sys_getrandom(Bytes.new(16)) rescue false }
```

Reply:
The memoization is irrelevant here, as this is called just once by class invariant.